### PR TITLE
Align scalyr buckets2

### PR DESCRIPTION
**Pull request by vetinari**
same as #271 with new remote repo and missing tests added

### CODE REVIEW

File: zmon_worker_monitor/builtins/plugins/scalyr.py [hunk 1]
```diff
@@ -124,20 +125,28 @@ def facets(self, filter, field, max_count=5, minutes=30, prio='low'):
...
         return j
 
-    def timeseries(self, filter, function='count', minutes=30, buckets=1, prio='low'):
+    def timeseries(self, filter, function='count', minutes=30, buckets=1, prio='low', align=0):
+        startTime = str(minutes) + 'm'
+        endTime = None
+        if align != 0:
+            cur_time = int(time.time())  # this assumes the worker is running with UTC time
+            endTime = cur_time - (cur_time % align)
+            startTime = endTime - (minutes * 60)
```
Comment (mohabusama) [inline zmon_worker_monitor/builtins/plugins/scalyr.py]: pep8 ``startTime`` -> ``start_time``

File: zmon_worker_monitor/builtins/plugins/scalyr.py [hunk 2]
```diff
@@ -124,20 +125,28 @@ def facets(self, filter, field, max_count=5, minutes=30, prio='low'):
         j = r.json()
         return j
 
-    def timeseries(self, filter, function='count', minutes=30, buckets=1, prio='low'):
+    def timeseries(self, filter, function='count', minutes=30, buckets=1, prio='low', align=0):
```
Comment (mohabusama) [inline zmon_worker_monitor/builtins/plugins/scalyr.py]: @vetinari let's switch this to align=30 by default and see what happens in longer periods. I would rather abstract this part from the User until they really hit a problem with it.

**Comment by vetinari** [comment]
:+1:

**Comment by mohabusama** [comment]
👍

**Pull request merged**